test/test.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
-const t = require('tap')
+const t = require('node:test')
+const assert = require('node:assert')
```

Comment:
remove `node:assert` in favor of test context assert [ref](https://nodejs.org/api/test.html#contextassert)